libraries/botbuilder-core/botbuilder/core/adapters/test_adapter.py

```diff
@@ -241,7 +241,9 @@ def id_mapper(activity):
         return await self.run_pipeline(context, self.logic)
 
     def get_next_activity(self) -> Activity:
-        return self.activity_buffer.pop(0)
+        if len(self.activity_buffer) > 0:
+            return self.activity_buffer.pop(0)
+        return None

```

Comment:
Do we have a test for this scenario? i.e. where buffer is empty

Reply:
I changed that to match the version in C#. It comes up during the unit test for skill dialog. Since the activity is sent to the skill it doesn't end up in the queue.